modules/perception/camera/app/lane_camera_perception.cc

```diff
@@ -0,0 +1,239 @@
+    auto lane_detector_param =
+        lane_param.lane_detector_param();
+    auto lane_detector_plugin_param = lane_detector_param.plugin_param();
+    lane_detector_init_options.conf_file =
+        lane_detector_plugin_param.config_file();
+    lane_detector_init_options.root_dir =
+        GetAbsolutePath(work_root, lane_detector_plugin_param.root_dir());
+    lane_detector_init_options.gpu_id = perception_param_.gpu_id();
+    model = common::SensorManager::Instance()->GetUndistortCameraModel(
+            lane_detector_param.camera_name());
+    auto pinhole = static_cast<base::PinholeCameraModel *> (model.get());
+    name_intrinsic_map_.insert(std::pair<std::string, Eigen::Matrix3f>(
+        lane_detector_param.camera_name(), pinhole->get_intrinsic_params()));
+    lane_detector_init_options.base_camera_model = model;
+    AINFO << "lane_detector_name: " << lane_detector_plugin_param.name();
+    lane_detector_.reset(BaseLaneDetectorRegisterer::GetInstanceByName(
+        lane_detector_plugin_param.name()));
+    CHECK(lane_detector_ != nullptr);
+    CHECK(lane_detector_->Init(lane_detector_init_options))
+    << "Failed to init " << lane_detector_plugin_param.name();
+    AINFO << "detector: " << lane_detector_->Name();
+
+    //  initialize lane postprocessor
+    auto lane_postprocessor_param =
+        perception_param_.lane_param().lane_postprocessor_param();
+    LanePostprocessorInitOptions postprocessor_init_options;
+    postprocessor_init_options.detect_config_root =
+        GetAbsolutePath(work_root, lane_detector_plugin_param.root_dir());
+    postprocessor_init_options.detect_config_name =
+        lane_detector_plugin_param.config_file();
+    postprocessor_init_options.root_dir =
+        GetAbsolutePath(work_root, lane_postprocessor_param.root_dir());
+    postprocessor_init_options.conf_file =
+        lane_postprocessor_param.config_file();
+    lane_postprocessor_.reset(
+        BaseLanePostprocessorRegisterer::GetInstanceByName(
+            lane_postprocessor_param.name()));
+    CHECK(lane_postprocessor_ != nullptr);
+    CHECK(lane_postprocessor_->Init(postprocessor_init_options))
+    << "Failed to init " << lane_postprocessor_param.name();
+    AINFO << "lane_postprocessor: " << lane_postprocessor_->Name();
+  }
+}
+
+void LaneCameraPerception::InitCalibrationService(
+            const std::string &work_root,
+            const base::BaseCameraModelPtr model,
+            const app::PerceptionParam &perception_param) {
+  // Init calibration service
+  CHECK(perception_param.has_calibration_service_param())
+  << "Failed to include calibration_service_param";
+  {
+    auto calibration_service_param =
+                      perception_param.calibration_service_param();
+    CalibrationServiceInitOptions calibration_service_init_options;
+    calibration_service_init_options.calibrator_working_sensor_name
+                    = lane_calibration_working_sensor_name_;
```

Comment:
ditto. And put "=" in previous line. Please also fix all other places.

Reply:
done